CONTRIBUTING.md

```diff
@@ -1,109 +1,62 @@
-
-### D6 - Regular Audits
-Following good programming practices is a way to reduce the risk of vulnerabilities, but professional code audits are still needed. We will perform regular code audits on major releases, and hire security professionals to provide independent review.
-
-## Style Guidelines
-
-The design guidelines have quite a high abstraction level. These style guidelines are more concrete and easier to apply, and also more opinionated.
-
-### General
-
-#### G0 - Default to Solidity's official style guide.
-
-Follow the official Solidity style guide: https://solidity.readthedocs.io/en/latest/style-guide.html
-
-#### G1 - No Magic Constants
-
-Avoid constants in the code as much as possible. Magic strings are also magic constants.
-
-#### G2 - Code that Fails Early
-
-We ask our code to fail as soon as possible when an unexpected input was provided or unexpected state was found.
-
-#### G3 - Internal Amounts Must be Signed Integers and Represent the Smallest Units.
-
-Avoid representation errors by always dealing with weis when handling ether. GUIs can convert to more human-friendly representations. Use Signed Integers (int) to prevent underflow problems.
-
-
-### Testing
-
-#### T1 - Tests Must be Written Elegantly
-
-Style guidelines are not relaxed for tests. Tests are a good way to show how to use the library, and maintaining them is extremely necessary.
-
-Don't write long tests, write helper functions to make them be as short and concise as possible (they should take just a few lines each), and use good variable names.
-
-#### T2 - Tests Must not be Random
-
-Inputs for tests should not be generated randomly. Accounts used to create test contracts are an exception, those can be random. Also, the type and structure of outputs should be checked.
-
-
-### Documentation
-
-TODO
-
-## Pull Request Workflow
+```
+cd my-zeppelin-solidity-fork
+git fetch upstream
+git checkout development
+git pull --rebase upstream development
+```
 
-Our workflow is based on GitHub's pull requests. We use feature branches, prepended with: `test`, `feature`, `fix`, `refactor`, or `remove` according to the change the branch introduces. Some examples for such branches are:
-```sh
-git checkout -b test/some-module
-git checkout -b feature/some-new-stuff
-git checkout -b fix/some-bug
-git checkout -b remove/some-file
+2) Branch out from `development` into `fix/some-bug-#123`:
+(Postfixing #123 will associate your PR with the issue #123 and make everyone's life easier =D)
+```
+git checkout -b fix/some-bug-#123
 ```
 
-If your branch is planned to fix an open issue, postfix your branch name with the issue number like `fix/some-bug-#123`.
+3) Make your changes, add your files, commit and push to your fork.
 
-We expect pull requests to be rebased to the master branch before merging:
-```sh
-git remote add zep git@github.com:OpenZeppelin/zeppelin-solidity.git
-git pull --rebase zep master
+```
+git add -A
```

Comment:
I think I'd prefer to discourage `git add --all`. Not sure though... It's just that explicitly listing the files feels better to me.